scripts/limonade/price-config.sample.yaml

```diff
@@ -0,0 +1,6 @@
+- alias: SOL
```

Comment:
can you include some notes in the README about cloning this? and also any details about where to grab heremes prices, the new args here, etc?

Reply:
updated readme